src/pages/Designer/Designer.js

```diff
@@ -25,35 +25,111 @@ const Designer = () => (
+                        Designer
+                      </Heading>
+                      <Paragraph
+                        margin={{ top: 'none' }}
+                        size="xxlarge"
+                        color="darkGrey"
+                      >
+                        Grommet Designer is a tool used to build expiriences
+                        with Grommet components; then publish and share your
+                        ideas with a simple wysiwg interface
+                      </Paragraph>
+                      <Box
+                        round="medium"
+                        background="buttonYellow"
+                        onClick={() => {}}
+                        width="small"
+                        height="xxsmall"
+                        justify="center"
+                        margin={{ bottom: 'small' }}
+                      >
+                        <Text weight="bold" alignSelf="center" color="black">
+                          Open Designer
+                        </Text>
+                      </Box>
+                    </Box>
+                  </Box>
+                </Stack>
+              </Box>
+              <Box
+                direction="column"
+                gap="medium"
+                margin={{ left: 'xlarge', bottom: 'large' }}
               >
-                Designer
-              </Heading>
+                <Box direction="row" margin={{ left: 'xlarge' }} gap="xlarge">
+                  <Camera opacity="0.8" size="large" color="ImagerOrange" />
+                  <Configure opacity="0.9" size="large" color="ThemerOrange" />
+                </Box>
+                <Box margin={{ left: 'xlarge' }} direction="row">
+                  <Box width="xxsmall"></Box>
+                  <CloudUpload
+                    opacity="0.7"
+                    size="large"
+                    color="PublisherPink"
+                  />
+                </Box>
+                <Box margin={{ left: 'xlarge' }} direction="row">
+                  <Box width="xsmall"></Box>
+                  <Selection opacity="0.5" size="large" color="SlidesBlue" />
+                </Box>
+                <Box margin={{ left: 'xlarge' }}>
+                  <BarChart opacity="0.3" size="large" color="TabularGreen" />
+                </Box>
+              </Box>
             </Box>
-            <Box height="gradient1" background="gradient"></Box>
-            <Box alignSelf="center" margin={{ left: 'xlarge' }} width="large">
+          ) : (
+            <Box margin={{ top: 'xlarge' }}>
+              <Box margin={{ left: 'xlarge' }}>
+                <Text> Grommet Tools </Text>
+                <Heading
+                  level={1}
+                  size="xlarge"
+                  margin={{ top: 'none', bottom: 'medium' }}
+                >
+                  Designer
+                </Heading>
+              </Box>
+              <Box height="gradient1" background="gradient" />
```

Comment:
I think you don't need to use `height` and you can remove the definition for "gradient1" on the theme.
You can just use `pad="small"` instead or other t-shirt size.